### PR TITLE
Make Stormwind vassal feudal

### PR DESCRIPTION
**Pull request by IronyOwl**
## Changelog:
- Count Elendur, a Stormwind vassal, is now feudal rather than tribal
- Stonecairn is now ruled by an independent Murloc dynasty

### CODE REVIEW

**Review by Remerod — APPROVED** [review]
He seems Feudal enough for me in "The Invasion from Draenor" bookmark ;)

<details>
<summary>Click to expand</summary>

image
 In reign of chaos to
image
And also in the Frozen Throne Bookmark :)
image

</details>

File: history/titles/00_k_stormwind.txt [hunk 1]
```diff
@@ -315,33 +315,22 @@ c_azora = {
...
+	396.5.6={holder=murloc128}
+	465.7.8={holder=murloc129}
+	501.7.13={holder=murloc130}
+	539.9.15={holder=murloc131}
 	# Conquered by the Horde
 	587.10.1={holder=10050} #Orgrim[2001]
-	# Retaken by Stormwind
-	590.11.1={holder=310240} #Elendur[patron]
+	# Horde broken, murlocs move back in
+	590.11.2={holder=murloc131}
```
Comment (zumbak04) [inline history/titles/00_k_stormwind.txt:333]: `murloc131` dies in `596.3.17` so you need to pass the title to his heir.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: history/titles/00_k_stormwind.txt (hunk 1 above)
Comment (IronyOwl) [reply history/titles/00_k_stormwind.txt:333]: Added that in.

**Comment by robmart** [comment]
@IronyOwl You forgot to re-request review from Zum.

**Review by zumbak04 — APPROVED** [review]
LGTM! Good job!

**Pull request merged**